src/main/java/com/intuit/apl/engine/ParsingRules.java

```diff
@@ -275,37 +279,13 @@ private void loadRuleFiles(String[] fileNames, InputStream inputStream, RulePack
    *
    * @param fileNames File names of the apl files Parsing PolicyFile files
    */
-  // void parseRuleFiles(String[] fileNames, InputStream fileRepository){
-  // Yaml yaml = new Yaml();
-  // RulePackage rulePackage = new RulePackage();
-  //
-  // if(null == ruleDefinitions) {
-  // Comparator<RuleDefinition> priorityComparator = new Comparator<RuleDefinition>() {
-  // @Override
-  // public int compare(RuleDefinition rd1, RuleDefinition rd2) {
-  // return rd2.getSalience() - rd1.getSalience();
-  // }
-  // };
-  // ruleDefinitions = new PriorityQueue<>(priorityComparator);
-  // SpelParserConfiguration config = new SpelParserConfiguration(SpelCompilerMode.IMMEDIATE,
-  // this.getClass().getClassLoader());
-  // parser = new SpelExpressionParser(config);
-  // }
-  // loadRuleFiles(fileNames, inputStream, rulePackage, yaml);
-  // }
-  //
   void parseRuleFiles(String[] fileNames, InputStream inputStream,
       PolicyRepository policyRepository) {
-    Yaml yaml = new Yaml(new Constructor(HashMap.class));
+    Yaml yaml = new Yaml();
```

Comment:
Do you wish to add the taginspector for not parsing malicious content ?

Reply:
By default, it doesn’t allow global tags to set thereby fixing that security vulnerability. This [article](https://www.veracode.com/blog/research/resolving-cve-2022-1471-snakeyaml-20-release-0) also talks about it.